packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -248,11 +248,12 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     maxDominantSpeakers: MAX_AUDIO_DOMINANT_SPEAKERS
   });
 
-  /* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
-  const localCameraCycleButton = (localVideoCameraCycleButtonProps): JSX.Element => {
-    const ariaDescription = _formatString(strings.localVideoSelectedDescription, {
-      cameraName: localVideoCameraCycleButtonProps.selectedCamera.name
-    });
+  const LocalCameraCycleButton = useCallback((): JSX.Element => {
+    const ariaDescription =
+      localVideoCameraCycleButtonProps?.selectedCamera &&
+      _formatString(strings.localVideoSelectedDescription, {
+        cameraName: localVideoCameraCycleButtonProps.selectedCamera.name
+      });
     return (
       <Stack horizontalAlign="end">
         {showCameraSwitcherInLocalPreview &&
```

Comment:
Not for this PR: We should ideally not render the `Stack` when all these conditions are not met (instead of rendering an empty `Stack`).